synapse/lib/storm.py

```diff
@@ -2926,3 +2926,62 @@ async def execStormCmd(self, runt, genr):
 
                     async for node in self.iterEdgeNodes(verb, idenset, n2):
                         yield node, _path.fork(node)
+
+class DelEdgesCmd(Cmd):
+    '''
+    Bulk delete light edges from input nodes.
+
+    Examples:
+
+        # Delete all "foo" light edges from an inet:ipv4
+        inet:ipv4=1.2.3.4 | deledges foo
+
+        # Delete light edges with any verb from a node
+        inet:ipv4=1.2.3.4 | deledges *
+
+        # Delete all "foo" light edges to an inet:ipv4
+        inet:ipv4=1.2.3.4 | deledges foo --n2
+    '''
+    name = 'deledges'
```

Comment:
Lets do a quick goback about potentially namspacing this command name

Reply:
Updated to `edge.del`